### PR TITLE
Add admin::company show and destroy actions Closes #30

### PR DESCRIPTION
**Pull request by laurawhalin**
added routes, controller and view for Admin::Companies, with the ability to view and destroy companies. added tests

screen shot 2015-04-23 at 3 37 55 pm

### CODE REVIEW

**Comment by cluhring** [comment]
Anyone can screen shot passing tests.  Makes me think you're trying to hide something Laura.

File: app/controllers/admin/companies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Admin::CompaniesController < ApplicationController
+  def show
```
Comment (cluhring) [inline app/controllers/admin/companies_controller.rb:2]: nitpicking, i'd love to see a couple controller tests.

File: spec/features/admin_dashboard_spec.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    end
+
+    it "can click to view details of a company" do
+      companies = create_list(:company, 10, status: "uncontacted")
+      login
+
+      within(".uncontacted-companies") do
+        click_link(companies.first.organization, match: :first)
+      end
+      expect(current_path).to eq(admin_company_path(companies.first.id))
```
Comment (cluhring) [inline spec/features/admin_dashboard_spec.rb]: nice tests, Laura!

File: app/controllers/admin/companies_controller.rb (hunk 1 above)
Comment (laurawhalin) [inline app/controllers/admin/companies_controller.rb:2]: Done! Thanks for asking me to do that!

File: spec/controllers/admin_companies_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    it "deletes the company" do
+      expect{ delete :destroy, id: company.id }.to change(Company,:count).by(-1)
+    end
+
+    it "redirects back to admin dashboard index after deletion" do
+      delete :destroy, id: company.id
+      expect(response).to redirect_to(admin_dashboard_index_path)
+    end
+  end
+end
```
Comment (bhargavisatpathy) [inline spec/controllers/admin_companies_controller_spec.rb:32]: Great! you did all these...we almost forgot to do that :)

File: spec/features/admin_dashboard_spec.rb [hunk 4]
```diff
@@ -37,9 +36,19 @@ def login
...
+    end
+
+    it "can click to view details of a company" do
+      companies = create_list(:company, 10, status: "uncontacted")
+      login
+
+      within(".uncontacted-companies") do
+        click_link(companies.first.organization, match: :first)
       end
+      expect(current_path).to eq(admin_company_path(companies.first.id))
```
Comment (bhargavisatpathy) [inline spec/features/admin_dashboard_spec.rb]: Looks good to me :)

File: app/views/admin/dashboard/index.html.erb [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 <div class="uncontacted-companies">
   <% @uncontacted_companies.each do |company| %>
-    <%= company.organization %>
+    <%= link_to company.organization, admin_company_path(company.id) %>
   <% end %>
 </div>
 <div class="contacted-companies">
   <% @contacted_companies.each do |company| %>
-    <%= company.organization %>
+    <%= link_to company.organization, admin_company_path(company.id) %>
```
Comment (oorion) [inline app/views/admin/dashboard/index.html.erb:25]: We may need to add two more links for confirmed and dead companies now that my presenter class pr is merged.

Comment (laurawhalin) [inline app/views/admin/dashboard/index.html.erb:25]: Already done! Thanks!

**Pull request merged**